lualib/resty/events/callback.lua

```diff
@@ -110,9 +110,18 @@ function _M:do_event(d)
     local event  = d.event
     local data   = d.data
     local wid    = d.wid
+    local time   = d.time
+
+    ngx.update_time()
+    local now = ngx.now()
+
+    if time then
+      log(DEBUG, "worker-events: receive event; source=", source,
```

Comment:
Could you add the payload to other logs as well?

Reply:
done.